server/src/main/java/org/apache/druid/server/coordinator/duty/EmitClusterStatsAndMetrics.java

```diff
@@ -234,36 +218,40 @@ private void emitStatsForHistoricalManagementDuties(DruidCluster cluster, Coordi
         }
     );
 
+    // Log load queue status of all replication or broadcast targets
     log.info("Load Queues:");
-    for (Iterable<ServerHolder> serverHolders : cluster.getSortedHistoricalsByTier()) {
```

Comment:
Change summary: Report queue status not just of historicals but all replication or broadcast targets.